tools/linter/install/clang_tidy.py

```diff
@@ -0,0 +1,18 @@
+import os
+from download_bin import download, PYTORCH_ROOT, HASH_PATH  # type: ignore[import]
+
+PLATFORM_TO_URL = {
+    "Linux": "https://oss-clang-format.s3.us-east-2.amazonaws.com/linux64/clang-tidy",
+}
+
+PLATFORM_TO_HASH = {
+    "Linux": os.path.join(HASH_PATH, "clang-tidy-linux64"),
```

Comment:
I like the files approach because then its easy to write tools that automate updating the hash upon a new release (simply commit a modified version of the hash to the repo). If we inline it, it might make it a little more complex to do this (one approach would be to regex on the dict structure and update the hash, but then this makes our script coupled to the way we represent the OS to hash mapping)